packages/contracts-ecosystem/src/Create2Factory.sol

```diff
@@ -0,0 +1,25 @@
+pragma solidity ^0.8.24;
+
+import {StdUtils} from "forge-std/StdUtils.sol";
+
+contract Create2Factory {
```

Comment:
this is just for an example we should never actually use this contract without adding real safety/auth logic to it